### ISSUE TITLE
Find and fix incompatibilities in Peatio's JSON RPC with BitGo

### ISSUE THREAD

**Issue by yivo** [openware/peatio#339]

**Comment by ysv** [comment]
1. listtransactions 
 ```
CoinRPC[coin.code.to_sym].listtransactions('payment', coin.code.xrp? ? 100 : 1000)
CoinRPC::JSONRPCError: {"code"=>-32603, "message"=>"Error: accounts not supported - use blank account only"}
```
https://github.com/rubykube/peatio/blob/4cbe0474ae752e1aedcea9757d6de37a78accfdd/lib/daemons/coins.rb#L9

2. detail[:account] != 'payment'

https://github.com/rubykube/peatio/blob/4cbe0474ae752e1aedcea9757d6de37a78accfdd/app/models/worker/deposit_coin.rb#L20

**Comment by yivo** [comment]
Closed in #355.
